docs/UserGuide.md

```diff
@@ -32,8 +32,8 @@ This guide serves to provide a reference for first-time users to get familiar wi
 
 1. Copy `ZooKeep.jar` to the folder you want to use as the _home folder_ for the app.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Double-click the file to start the app. The GUI similar to *Figure 1* below should appear in a few seconds. Note how the app contains some sample data.<br>
+   ![Ui](images/Ui.png)*Figure 1: GUI upon starting ZooKeep*
```

Comment:
Small nitpick: Maybe can add shown in the label description i.e. GUI "shown" upon...